apps/site/wrangler.jsonc

```diff
@@ -6,6 +6,7 @@
   "account_id": "8ed4d03ac99f77561d0e8c9cbcc76cb6",
   "minify": true,
   "keep_names": false,
+  "find_additional_modules": true,
```

Comment:
This should not be needed as .wasm file will be added by default.
See https://developers.cloudflare.com/workers/wrangler/bundling/#including-non-javascript-modules